.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2.
+3.
+4.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment (please complete the following information):**
+ - **Collection version** (e.g. `helm ls -n sumologic`):
+ - **Kubernetes version** (e.g. `kubectl version`):
+ - **OS** (e.g. from /etc/os-release):
+ - **Kernel** (e.g. `uname -a`):
```

Comment:
Do we need that?

Reply:
If we suppose that the issue may be related to k8s then kernel version may be helpful. Some k8s defects depends on kernel version.

Reply:
So is it about `k8s` machines `OS` and `Kernel` or client machine?

Reply:
I modified the template, I hope it looks better now.